examples/api/lib/material/context_menu/anywhere.dart

```diff
@@ -0,0 +1,154 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This sample demonstrates allowing a context menu to be shown in a widget
+// subtree in response to user gestures.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  void _showDialog (BuildContext context) {
+    Navigator.of(context).push(
+      DialogRoute<void>(
+        context: context,
+        builder: (BuildContext context) =>
+          const AlertDialog(title: Text('You clicked print!')),
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        appBar: AppBar(
+          title: const Text('Context menu in SelectionArea'),
+        ),
+        body: _ContextMenuRegion(
+          contextMenuBuilder: (BuildContext context, Offset primaryAnchor, [Offset? secondaryAnchor]) {
+            // The custom context menu will look like the default context menu
+            // on the current platform, but it will have only a single 'Print'
+            // button.
+            return DefaultTextSelectionToolbar(
+              primaryAnchor: primaryAnchor,
+              secondaryAnchor: secondaryAnchor,
+              buttonItems: <ContextMenuButtonItem>[
```

Comment:
can `buttonItems` be called `children`, to keep in line with what most container widgets in Flutter use? And is it necessary to use a specific type of item, or could you use any `Widget` instead?

Reply:
I allow either/or, Widgets or ContextMenuButtonItems, mainly for convenience.  @Renzo-Olivares was also confused by why I did this.  I just added a new [appendix](https://docs.google.com/document/d/1F1itGgbSS3Cbq70hScSCNriDoux1sMGF9LHZsMdvU74/edit?resourcekey=0-RkHGwhe-tcoTGlaQyclPtQ#heading=h.lbwwfocsw4n5) to my design doc to better explain this, along with some examples.  Let me know if this pattern still doesn't make sense or if I can improve it.